chpl/chpl-resources/src/main/resources-production/jobs.xml

```diff
@@ -27,6 +27,111 @@
                 </entry>
             </job-data-map>
         </job>
+
+        <!-- Questionable URL Data Collector -->
+        <!-- URL Status Data Collector -->
+        <!-- Job used to change status of a set listings -->
+        <job>
```

Comment:
Why are all of these in this PR? Shouldn't they either be in a separate PR, or if already on staging, not in a PR at all?

Reply:
Just to follow up on this - this file as it exists on staging right now has 2 jobs in it.